requirements.txt

```diff
@@ -12,3 +12,9 @@ Flask-SQLAlchemy==2.3.2
 colorama==0.3.9
 colour==0.1.5
 psycopg2-binary==2.7.5
+webargs==4.0.0
+Werkzeug==0.14.1
+pytest==3.7.2
+pytest-runner==4.2
```

Comment:
Why include test related requirements as base requirements? Besides that I think that there is no tests on tag project.

Reply:
Now there are test and all is broken but I suppose that we want to do a tests

Reply:
What I mean is that requirements.txt is the file used by a user to deploy tag service. A normal user (not a developer) doesn't want to install dependencies only related to run tests.

So, my proposal is to move test related dependencies to a new file (e.g. requirements-test.txt).